app/src/main/res/values/styles.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools">
 
-    <style name="BaseAppTheme" parent="Theme.MaterialComponents.Light.NoActionBar">
+    <style name="BaseAppTheme" parent="Theme.MaterialComponents.Light.NoActionBar.Bridge">
```

Comment:
This seems like a very blunt way of fixing the issue. The `Bridge` themes inherit from the old `AppCompat` themes, which we should be trying to get away from, instead of going back to.

Can we just override the button style itself, with something like this:
```
<item name="materialButtonStyle">@style/Widget.MaterialComponents.Button.TextButton</item>`
```
(put it in the `BottomSheetDialogLight` and `BottomSheetDialogDark` styles)

Reply:
Hi @dbrant, thanks for providing the suggestion and example.

I tried to put `materialButtonStyle` item to `BottomSheetDialogLight` and `BottomSheetDialogDark`, but nothing happened. I tested it on an API 23 emulator.

Reply:
@cooltey Try also adding the same `materialButtonStyle` item to the top-level `BaseAppTheme`.  (and verify that no other buttons are affected by accident)

Reply:
Will do, thanks!

Reply:
Done by adding `materialButtonStyle` to `BaseAppTheme` and tested it on API 23, API 24 and API 28, every button looks good.